frontend/src/store/selectors.ts

```diff
@@ -175,25 +176,39 @@ export const selectedSubExperimentsQuery = selector<SubExperimentWithMeasurement
   key: "SelectedSubExperiments",
   get: ({ get }) => {
     const ids = Array.from(get(selectedSubExperimentsIdsAtom).values());
-    return get(waitForAll(ids.map(subId => subExperimentsWithMeasurementsQuery(subId))));
+    return get(waitForAll(ids.map(subId => subExperimentWithMeasurementsMap(subId))));
+  },
+});
+
+export const selectedSubExperimentsMetaQuery = selector<SubExperimentDto[]>({
+  key: "SelectedSubExperimentsMeta",
+  get: ({ get }) => {
+    const ids = Array.from(get(selectedSubExperimentsIdsAtom).values());
+    return get(waitForAll(ids.map(subId => subExperimentsMetaMap(subId))));
   },
 });
 
 export const selectedSubExperimentsIC50Query = selector<ResultDto<number>>({
   key: "SelectedSubExperimentsIC50",
   get: ({ get }) => {
-    const subs = get(selectedSubExperimentsQuery);
-    const ic50Sum = subs.reduce(
-      (acc: ResultDto<number>, x) =>
-        "Left" in acc
-          ? acc
-          : "Left" in x.meta.item.result
-          ? { Left: x.meta.item.result.Left }
-          : { Right: acc.Right + x.meta.item.result.Right[2] },
-      { Right: 0 }
-    );
-    if ("Left" in ic50Sum) return ic50Sum;
-    return { Right: ic50Sum.Right / subs.length };
+    return evalMeanIC50(get(selectedSubExperimentsMetaQuery).map(x => x.item.result));
```

Comment:
The issue says:
> Somewhere in Dashboard we show mean IC50 for all primary sub-experiments of all experiments where at least one sub-experiment is selected. I think that's what they want, not mean for all selected sub-experiments.

Currently we calculate average for all sub-experiments shown on plot, but I think they want each selected experiment to have equal weight, even if multiple sub-experiments are selected. They say:
> Необходим расчет среднего значения IC50 при выборе нескольких экспериментов

It's formulated imprecisely, so maybe I am wrong. Maybe it's better to have the following logic: if multiple sub-experiments are chosen from one experiment, we calculate average for them and then we take average of all these "averages". So if you have `se1` and `se2` from one experiment and `se3` for another experiment, we can return `((se1 + se2) / 2 + se3) / 2`. I don't insist on doing that and I don't know what they want. If you think I am saying nonsense, feel free to ignore. In general please try to maximize usefulness of this value divided by complexity of implementing it.

Reply:
>So if you have se1 and se2 from one experiment and se3 for another experiment, we can return ((se1 + se2) / 2 + se3) / 2

Ok, maybe it makes sense... it's not hard to implement